includes/Template/Woo-Product-Gallery/default.php

```diff
@@ -148,7 +148,7 @@ class="eael-product-gallery-open-popup open-popup-link">
                                 <?php } ?>
 			                    <?php if( $should_print_link ){?>
                                     <li class="view-details"><?php echo '<a href="' . $product->get_permalink
-                                        () . '"><i class="fas fa-link"></i></a>'; ?></li>
+                                        () . '" aria-label="View Details about '. esc_attr( $product->get_title() ) .'"><i class="fas fa-link"></i></a>'; ?></li>
```

Comment:
add spaces where applicable.
```
() . '" aria-label="View Details about ' . esc_attr( $product->get_title() ) . '"><i class="fas fa-link"></i></a>'; ?></li>
```

Reply:
We shouldn't use $product->get_permalink and () in two lines.
It should be on 1 line.
```
$product->get_permalink()
```